rpc/client/client.go

```diff
@@ -267,8 +267,38 @@ func (c *Client) GenesisChunked(context context.Context, id uint) (*ctypes.Resul
 }
 
 func (c *Client) BlockchainInfo(ctx context.Context, minHeight, maxHeight int64) (*ctypes.ResultBlockchainInfo, error) {
-	// needs block store
-	panic("BlockchainInfo - not implemented!")
+	const limit int64 = 20
+	minHeight, maxHeight, err := filterMinMax(
+		0,
```

Comment:
We possibly should implement something like [this](https://github.com/tendermint/tendermint/blob/4ba5a053dea17f60dfbf0b68f6f477bbfc598b85/internal/store/store.go#L44) in our store right ?